com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/QueryOperators.java

```diff
@@ -40,5 +40,6 @@ public interface QueryOperators {
 	public static final String VALRULE    = "$valrule";
 	public static final String CONTEXT    = "$context";
 	public static final String REPORTTYPE = "$report_type";
+	public static final String INCLUDE_INACTIVE   = "includeinactive";
```

Comment:
until now the standard on these variables doesn't include underscore, better to name it INCLUDEINACTIVE